### ISSUE TITLE
pascal_voc_evaluator.py how to get the FN

### ISSUE THREAD

**Issue by tjbe2021** [rafaelpadilla/review_object_detection_metrics#34]
Hi there, would you be able to help with how to get the False Negative number? Thanks.

**Comment by rafaelpadilla** [comment]
Hi @tjbe2021,

You could calculate the FN per image by looking at [this](https://github.com/rafaelpadilla/review_object_detection_metrics/blob/main/src/evaluators/pascal_voc_evaluator.py#L134) dictionary.

As you have the recall and TP values, you can calculate FN with the recall equation:
![image](https://user-images.githubusercontent.com/31217453/111663147-4b963200-87ef-11eb-8624-f0732f1116d8.png)

**Comment by tjbe2021** [comment]
Thanks for the swift response, yes,  as I understand from that code base and the previous version,  we have got  'rec = acc_TP / npos', and according to the above equation if I subtract total_TP from the 'npos', I should be getting total FN? however, when I do this, I'm getting 0 as the result. although I can clearly see there are FNs on my detection. Any thoughts?

e.g. 'total_FN = npos - total_TP'

and when I do the followings, 

total_FN  =  (total_TP / recall ) - total_TP  

I get the results with total_fn: [nan nan nan nan nan nan nan nan nan nan nan]

Any thoughts on what I'm doing wrong in here? I tried summing the list of recall, but still gets the 'total_fn = nan'

I'm actually tweaking your earlier code, https://github.com/rafaelpadilla/Object-Detection-Metrics
